frontend/model/notifications/periodicNotifications.js

```diff
@@ -1,7 +1,6 @@
 'use strict'
 
 import sbp from '@sbp/sbp'
-import Vue from 'vue'
```

Comment:
Could you add a comment near the top of the file explaining:

- That this file is used in both the SW and the Browser UI
- That we do not import Vue for two reasons: (1) it's bulk that's not needed in the SW, and (2) what @SebinSong [said](https://github.com/okTurtles/group-income/pull/2459#discussion_r1902262722):

> I see multiple places where Vue.delete and Vue.set are replaced with pure Javascript logics here. I think it is fine as alreadyFired and lastRun maps are currently only used in this file and [1] there is no place in the app to re-render in response to these state updates, as far as I know.